### DISCUSSION TITLE
rancher-desktop fails to start on macOS with the Error Code 255

### DISCUSSION THREAD

**Discussion by MananTheCoder** [rancher-sandbox/rancher-desktop#4764] [Q&A]
Hi!
I just did a fresh download and installation of rancher-desktop 1.4.1 on my Mac OS Monterey 12.5.1. Whenever I try to start it, it throws the following error (Error code 255).
<img width="767" alt="Rancher Error Code 255" src="https://github.com/rancher-sandbox/rancher-desktop/assets/48885067/2ade6c98-1fbb-4d7b-b746-ea9b6bdc89de">

These are the contents of the logfile section in the above ss.
```
at ChildProcess.emit (node:events:390:28)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:290:12) {
  command: [
    '/Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/bin/kubectl',
    '--context',
    'rancher-desktop',
    'cluster-info'
  ]
}
```
Please suggest what could be going wrong here and how can we resolve it. I shall be happy to share any other info as may be required.

**Comment by jandubois** [comment]
Please try Rancher Desktop 1.8.1 and do a factory reset.

**Reply by MananTheCoder** [reply]
Can't find 1.8.1 on [Uno](https://uno.oraclecorp.com/uno/). The latest being shown is 1.7.0.
<img width="1039" alt="Screenshot 2023-05-26 at 3 04 44 PM" src="https://github.com/rancher-sandbox/rancher-desktop/assets/48885067/502ec121-5136-4bef-9c37-5799d4cb5560">

**Comment by jandubois** [comment]
The official releases are available from https://github.com/rancher-sandbox/rancher-desktop/releases
